src/strategies/index.ts

```diff
@@ -207,5 +208,6 @@ export default {
   'balancer-delegation': balancerDelegation,
   'infinityprotocol-liquidity-pools': infinityProtocolPools,
   'aave-governance-power': aaveGovernancePower,
-  cake
+  cake,
+  planetFinance
```

Comment:
The strategy name should use kebab case, i recommend to change to this:
```suggestion
  'planet-finance': planetFinance
```

Reply:
@ash-greninja-poke The strategy folder too. If you can change both the strategy will be ready to merge.

Reply:
@bonustrack i have made the changes can you please review it now?